.changeset/red-crabs-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-pantheon-decoupled-kit": minor
```

Comment:
I think the `create-pantheon-decoupled-kit` changeset is a patch version? maybe not. But either way there should be a minor version bump to `drupal-kit`.